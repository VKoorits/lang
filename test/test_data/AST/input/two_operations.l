x = ((a + b)*c - 1)/3 + 8_000_000.78 +* 3
