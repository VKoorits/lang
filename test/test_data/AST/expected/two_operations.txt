ERROR: this tokens can`t be near:
	OPERATION_TOKEN: +, info => 9
	OPERATION_TOKEN: *, info => 10
